.github/BOTMETA.yml

```diff
@@ -1755,6 +1759,12 @@ files:
     support: core
   $plugins/doc_fragments/aci.py: *aci
   $plugins/doc_fragments/acme.py: *crypto
+  $plugins/doc_fragments/checkpoint_commands.py:
```

Comment:
These look like modules, rather than docs fragments

Reply:
Location in core is lib/ansible/plugins/doc_fragments/checkpoint_commands.py